src/components/suggest/suggest_input.js

```diff
@@ -0,0 +1,137 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { EuiFilterButton } from '../filter_group';
+import { EuiFieldText } from '../form';
+import { EuiToolTip } from '../tool_tip';
+import { EuiIcon } from '../icon';
+import { EuiPopover, EuiInputPopover } from '../popover';
+
+export class EuiSuggestInput extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      value: '',
+      isPopoverOpen: false,
+    };
+  }
+
+  onFieldChange(e) {
+    this.setState({
+      value: e.target.value,
+      isPopoverOpen: e.target.value !== '' ? true : false,
+    });
+    this.props.sendValue(e.target.value);
+  }
+
+  closePopover() {
+    this.setState({
+      isPopoverOpen: false,
+    });
+  }
+
+  render() {
+    const {
+      className,
+      status,
+      append,
+      tooltipContent,
+      suggestions,
+      sendValue,
+      ...rest
+    } = this.props;
+
+    const statusMap = {
+      unsaved: {
+        icon: 'dot',
+        color: 'accent',
+        tooltip: "You've made changes that haven't been saved yet.",
```

Comment:
And this one could be more succinct too.
```suggestion
        tooltip: "Changes have not been saved.",
```